_Week5/group1/one_way_anova.do

```diff
@@ -2,9 +2,6 @@
 Week 5 Assignment
 Program .do file */
 
-* Explanation of ANOVA: https://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/BS704_HypothesisTesting-ANOVA/BS704_HypothesisTesting-Anova3.html
-* Note that this function does NOT run tests of assumptions needed for ANOVA; should run those tests in Stata before using this program
```

Comment:
I am unable to comment on the code below for some reason. My general comment is that it is very thoroughly commented and is readable and efficient. 

You may want to consider removing comments for basic self-explanatory functions like defining arguments, loading the dataset, and so on. Also, when loading the dataset, you can put the code in one line to make it a little more efficient.

Reply:
Oh no, sorry about the greyed-out lines that you can't add comments to! I'll ask about that in class today; I'm not able to comment on them either.

Great point about removing unnecessary comments. When you say "put the code in one line", you mean move the "clear" command above "sysuse" into the same line as "sysuse"? I think that's what you're getting at, but let me know if I've misinterpreted.